composer.json

```diff
@@ -20,7 +19,8 @@
     "szepeviktor/phpstan-wordpress": "^1.3",
     "axepress/wp-graphql-stubs": "^1.14",
     "axepress/wp-graphql-cs": "^2.0.0-beta",
-    "roave/security-advisories": "dev-latest"
+    "roave/security-advisories": "dev-latest",
+    "wp-graphql/wp-graphql": "^2.2"
```

Comment:
Do we need WPGraphQL as its own dev-dep? If you're trying to lint/typecheck against a specific version then wp-graphql-stubs without all the potential dependency conflicts.

Reply:
I will remove